monai/losses/dice.py

```diff
@@ -23,63 +23,82 @@
 @alias("dice", "Dice")
 class DiceLoss(_Loss):
     """
-    Multiclass dice loss. Input logits `pred` (BNHW[D] where N is number of classes) is compared with ground truth
```

Comment:
would be great to revise the docstring instead of removing it

Reply:
updated in the latest patch.
Thanks.